helm/cas-ciip-portal/Chart.yaml

```diff
@@ -12,3 +12,19 @@ dependencies:
     version: "3.3.0"
     repository: "https://codecentric.github.io/helm-charts/"
     condition: mailhog.enable
+  - name: cas-airflow-dag-trigger
+    version: 1.0.5
+    repository: https://bcgov.github.io/cas-airflow
+    alias: download-airflow-ciip-cert-renewal
+  - name: cas-airflow-dag-trigger
+    version: 1.0.5
+    repository: https://bcgov.github.io/cas-airflow
+    alias: download-airflow-ciip-deploy-db
+  - name: cas-airflow-dag-trigger
+    version: 1.0.5
+    repository: https://bcgov.github.io/cas-airflow
+    alias: airflow-ciip-cert-renewal
```

Comment:
The certificate renewal is executed on a schedule (daily), so it does not need to be triggered when deploying, You can remove this dependency

Reply:
We do need a manual trigger of the "issue" DAG, but only when the chart is being installed, so that should be a conditional dependency